BusinessLayer/Send_SuccessEmail.xaml

```diff
@@ -0,0 +1,111 @@
+<Activity mc:Ignorable="sap sap2010" x:Class="Send_SuccessEmail" xmlns="http://schemas.microsoft.com/netfx/2009/xaml/activities" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mva="clr-namespace:Microsoft.VisualBasic.Activities;assembly=System.Activities" xmlns:sap="http://schemas.microsoft.com/netfx/2009/xaml/activities/presentation" xmlns:sap2010="http://schemas.microsoft.com/netfx/2010/xaml/activities/presentation" xmlns:scg="clr-namespace:System.Collections.Generic;assembly=mscorlib" xmlns:sco="clr-namespace:System.Collections.ObjectModel;assembly=mscorlib" xmlns:ui="http://schemas.uipath.com/workflow/activities" xmlns:um="clr-namespace:UiPath.Mail;assembly=UiPath.Mail.Activities" xmlns:umab="clr-namespace:UiPath.Mail.Activities.Business;assembly=UiPath.Mail.Activities" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

```

Comment:
This should have been called Outlook_SendEmail. You should have a reusable component who can send either success or failure emails :)
Not suitable for BusinessLayer, move to ServiceLayer. Email, command line operations, shared folder operations, powershell operations all belong to ServiceLayer. 
Invoke send email worklow in Main too, with suitable arguments, and delete what you have there.